app/views/reviews/index.html.erb

```diff
@@ -61,9 +60,37 @@
         <td><%= link_to 'Destroy', review, method: :delete, data: { confirm: 'Are you sure?' } %></td>
       </tr>
     <% end %>
+    <button class="btn btn-primary" id="myBtn">Report Review</button>
+
+    <div id="myModal" class="modal">
+        <div class="modal-content">
+            <span class="close">&times;</span>
+            <div class ="header">
+              <h1>Report Review</h1>
+                <p>Please let us know why this review is being reported.</p>
+
+            </div>
+            
+            <div class="wrapper-class">
+              <input type="radio" id="radio1">
+                   <label for="radio1">It is spam or abusive.</label><br><br>
+              <input type="radio" id="radio2">
+                   <label for="radio2">It uses inappropriate or offensive language.</label><br><br>
+               <input type="radio" id="radio3">
+                    <label for="radio3">Other</label><br>
+            </div>
+            <input id="other_input" type="text"  placeholder="Please specify for 'Other'">
+
+            <button class="btn btn-primary w-80" id="submitReview">Submit Report</button>
```

Comment:
You have effectively described an HTML `<form>`. I would highly recommend you change this to a `<form>`, as it is:

1. More accessible to all users (such as those using screen readers)
2. Easier to test
3. Easier to maintain
4. Standardized with lots of documentation.